include/circt/Dialect/RTL/Combinatorial.td

```diff
@@ -143,13 +143,8 @@ def ExtractOp : RTLOp<"extract", [NoSideEffect]> {
     $input `from` $lowBit attr-dict `:` functional-type($input, $result)
   }];
 
-  let builders = [
-    OpBuilder<"OpBuilder &builder, OperationState &result, Type resultType, "
-              "Value input, unsigned lowBit">
-  ];
-
   let extraClassDeclaration = [{
-    unsigned getLowBit() { return lowBit().getZExtValue(); }
+    unsigned getLowBit() { return lowBit(); }
```

Comment:
Feel free to nuke this accessor, or I can take care of it in a follow up.

Reply:
Thanks, missed that one.